docs/pages/blog/nextgen-pickers.md

```diff
@@ -0,0 +1,130 @@
+---
+title: A fresh approach for date and time inputs
```

Comment:
I feel that it would be much better to tighten new feature announcements like this one to actual version releases. It makes it very clear which version includes this and gives a reason for people to share a specific release.